genesis/src/genesis_accounts.rs

```diff
@@ -614,6 +615,21 @@ fn add_stakes(
         .sum::<u64>()
 }
 
+pub const VALIDATOR_INFOS: &[ValidatorInfo] = &[ValidatorInfo {
+    name: "aurel@ethereum.ro",
+    node: "GeZ5PrJi9muVCJiJAaFBNGoCEdxGEqTp7L2BmT2WTTy1",
+    vote: "7ZdRx2EBYoRuPfyeoNbuHodMUXcAQRcC37MUw3kP6akn",
+    node_sol: 500.0,
+    commission: 50,
```

Comment:
0% commission for SLP1 no?

Reply:
open question, I think @ericlwilliams 

since no rewards for SLP1, commission is somewhat irrelevant

Reply:
Can we just set 0 so we can keep moving and revisit later if need.

Reply:
done